docs/intro/ff-ui/builder.md

```diff
@@ -0,0 +1,47 @@
+---
+slug: builder
+title: App Builder
+tags: []
+description: Explore the App Builder in FlutterFlow, featuring a comprehensive interface with four main sections. Navigation Menu, Toolbar, Canvas, and Properties Panel.
+sidebar_position: 0
+---
+
+# App Builder
+
+On opening the project, you'll see the App Builder, which consists of four main sections: [Navigation Menu](#navigation-menu), [Toolbar](#toolbar), [Canvas](#canvas-area), and [Properties panel](#properties-panel).
+
+![app-builder](imgs/builder.avif)
+
+## 1. Navigation Menu {#navigation-menu}
+
+The Navigation Menu, located on the left side of the builder, allows you to switch between various FlutterFlow features. These include designing the UI, managing databases, setting up API, adjusting app settings, adding integrations, and more.
+
+Here is a list of all the features accessible from the navigation menu:
+
+1. **Dashboard**: Manage projects, access account info, and FlutterFlow resources.
+2. **Widget Palette**: Access all widgets for your app.
+3. **Page Selector**: Manage pages and components, create folders as needed.
+4. **Widget Tree**: Get an overview of all widgets on a selected page.
+5. **Storyboard**: Visualize app's design and navigation.
+6. **Firestore**: Create collections and adjust Firestore-related settings.
+7. **Data Types**: Create custom data types for your app.
+8. **App Values**: Manage App State variables and Constants.
+9. **API Calls**: Define API calls.
+10. **Media Assets**: Upload assets for your app and team.
+11. **Custom Functions**: Add custom functionalities, widgets, and actions.
+12. **Cloud Functions**: Write and deploy cloud functions for Firebase.
+13. **Tests**: Add automated tests.
+14. **Theme settings**: Customize visual appearance.
+15. **Settings and Integrations**: Access app-related settings and integrations.
+
+## 2. Tool Bar {#toolbar}
```

Comment:
ToolBar won't have a space in between i think, please fix wherever applicable.

Reply:
Updated!